### PR TITLE
Create ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59255](https://cumulocity.atlassian.net/browse/MTM-59255)
Original PR: [6573](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6573)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Show assets and groups on list for edit type dashboard instance (#6351) [GRAFT][release/cd] (#6573)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Assets and groups are now shown when editing dashboard instances
```

File: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59255
+version: 1020.1.0
+---
+Show assets and groups on list for edit type dashboard instance (#6351) [GRAFT][release/cd] (#6573)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when editing a dashboard instance, the lists of assets and groups were not displayed, making it difficult for users to see which assets and groups were associated with the dashboard instance. With this change, the lists of assets and groups are now shown when editing a dashboard instance. This improves the user experience by providing more visibility into the configuration of the dashboard instance, allowing users to easily review and modify the associated assets and groups as needed.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:17]:
```suggestion
Dashboard manager is showing dashboards for both devices and groups/assets type dashboards (for the latter one they could be created when application options hideTypeDashboardForAssets=false), but when you click 'Edit', new modal was showing target assets only for devices dashboards, but not for groups/assets (for the groups/assets list was empty). Now, when you click Edit, list was always populated with devices and groups/assests.
```

**Review by rahultsag — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Assets and groups are now shown when editing dashboard instances
+product_area: Application enablement & solutions
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:7]: The description actually sounds more like a fix than a feature @jdreSAG .

Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:7]:
```suggestion
    label: Fix
```

**Review by rahultsag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Assets and groups are now shown when editing dashboard instances
+product_area: Application enablement & solutions
+change_type:
+  - value: change-QHu1GdukP
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md:6]:
```suggestion
  - value: change-VSkj2iV9m 
```

**Pull request merged**